jobboard/templates/jobboard/board.html

```diff
@@ -17,7 +40,13 @@ <h4>{{user_as_recruiter.company}}</h4>
     </div>
     <div class="col-10 text-center">
       <div class="row">
-        {% if user_is_student %}
+        {% if searched_name and not searched_area %}
```

Comment:
Maybe add a "title" key to the context dictionary, than you can handle all these ifs outside of the template

Reply:
Done